.isort.cfg

```diff
@@ -1,6 +0,0 @@
-[settings]
-multi_line_output=3
-include_trailing_comma=True
-force_grid_wrap=0
-use_parentheses=True
-line_length=79
```

Comment:
Lets get rid of `isort`. Ruff is already doing its own things, and minimizing potential linter issues is ideal. 

We'll need to remove it from the `pyproject.toml` and `.pre-commit-config.yaml`. It's already excluded from our GH actions.